installers/python-ci/radiasoft-download.sh

```diff
@@ -43,10 +44,11 @@ python_ci_main() {
             set -x
             cd '$d'
             export GITHUB_TOKEN='${GITHUB_TOKEN:-}'
-            if (( $p )); then
-                pip uninstall -y pykern >& /dev/null || true
-                pip install git+https://github.com/radiasoft/pykern.git
-            fi
+            for x in "${p[*]}"
+            do
+                pip uninstall -y \$x >& /dev/null || true
+                pip install git+https://github.com/radiasoft/\$x.git
```

Comment:
I think we will want the this:

`pip install git+https://'${GITHUB_TOKEN:+$GITHUB_TOKEN@}'/radiasoft/\$x.git`

The GITHUB_TOKEN avoids rate limiting issues we have run into in Actions.

Reply:
085af5b7bc7474e434c3f7ad7cc88ba32ed8cbb8